.github/workflows/ci.yml

```diff
@@ -179,4 +179,4 @@ jobs:
       - name: Test
         shell: bash
         working-directory: ${{ github.workspace }}/build
-        run: ctest --output-on-failure
+        run: ctest --output-on-failure || (echo "=== Server Log ===" && cat ./tests/netopeer2-server.log && echo "=== End of Server Log ===")
```

Comment:
After latest changes this is not that simple and the path depends on the specific test that failed.